src/components/MoneyRequestConfirmationList.js

```diff
@@ -267,6 +267,8 @@ function MoneyRequestConfirmationList(props) {
         return (props.hasSmartScanFailed && TransactionUtils.hasMissingSmartscanFields(transaction)) || (didConfirmSplit && TransactionUtils.areRequiredFieldsEmpty(transaction));
     }, [props.isEditingSplitBill, props.hasSmartScanFailed, transaction, didConfirmSplit]);
 
+    const isIOUMerchantPresent = props.isPolicyExpenseChat && !props.isScanRequest && !props.iouMerchant;
```

Comment:
```suggestion
    const isMerchantEmpty = props.isPolicyExpenseChat && !props.isScanRequest && !props.iouMerchant;
```

Reply:
"is this the cleanest way to add the merchant empty check to this form?"
I think yes, we need to check for an expense chat, but scanRequest should work as previously.

"also wont the reset of the money request remove the Request from merchant too?"
Hm, didn't get a question, don't we want to remove default "Request" text from the merchant?

Reply:
Ahh, okay, you mean that merchant field must be empty only for expense chats, correct?
@mountiny

Reply:
We sont want to remove it for IOU peer to peer request. Only for expense requests so group policies